src/main/java/minicraft/saveload/Load.java

```diff
@@ -238,6 +238,16 @@ private void loadGame(String filename) {
 			Settings.set("quests", Boolean.parseBoolean(data.remove(0)));
 			Settings.set("tutorials", Boolean.parseBoolean(data.remove(0)));
 		}
+
+		File zip = new File(location + "resources.zip");
```

Comment:
`location` has slash at the end. resources.zip lets the default resource packs world-bundle.

Reply:
Doesn't make sense

Reply:
World-bundle default resource pack replaces the original default resource pack so that the players could make world-bundle world with resources and don't have to make another external resource packs.

Reply:
That is pointless, and only complicates this already too complex system even more. Use the KISS principle, please.